src/core/Akka.API.Tests/CoreAPISpec.ApproveStreams.verified.txt

```diff
@@ -2280,6 +2292,7 @@ namespace Akka.Streams.Dsl
         public abstract Akka.Streams.Dsl.IFlow<T2, TMat> Via<T2, TMat2>(Akka.Streams.IGraph<Akka.Streams.FlowShape<TOut, T2>, TMat2> flow);
         public abstract Akka.Streams.Dsl.IFlow<T2, TMat3> ViaMaterialized<T2, TMat2, TMat3>(Akka.Streams.IGraph<Akka.Streams.FlowShape<TOut, T2>, TMat2> flow, System.Func<TMat, TMat2, TMat3> combine);
     }
+    public delegate void SwapPrevious<T>(T previousValue, T newValue);
```

Comment:
I created an explicit delegate type here in order to make it clear which value was the previous and which one was the new, since I personally had some confusion with it while I was working on the sample.